docs/reference/data-analysis/aggregations/search-aggregations-bucket-daterange-aggregation.md

```diff
@@ -30,6 +30,7 @@ POST /sales/_search?size=0
   }
 }
 ```
+% TEST[setup:sales s/now-10M/M/10-2015/]
```

Comment:
This should be `TEST[setup:sales s/now-10M\/M/10-2015/]` (one of the / should be escaped)

Reply:
Addressed via [2e570d1](https://github.com/elastic/elasticsearch/pull/124283/commits/2e570d1d9819de81d7a9018391f9665ab29e1ca1)